src/Controller/SkillsController.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Skills;
+use App\Form\SkillsType;
+use App\Repository\SkillsRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/skills")
+ */
+class SkillsController extends AbstractController
+{
+
+    public const CATEGORIES = [
+        'Droit et finance',
+        'Marketing et numérique',
+        'Commerce et logistique',
+        'Administratif',
+        'Autres',
+    ];
```

Comment:
Assuming this is the Skills categories, this constant should probably be in the Skill class

Reply:
After the last meeting with the customer we agreed that the categories would be fixed, so we decided to simplify our database by deleting the category table and writing them in the code.
I'm not sure if I translated the remark correctly sorry

Reply:
ooooh ok, I think I understand